src/main.zig

```diff
@@ -292,10 +292,21 @@ fn openCommand() void {
 }
 fn takeBackgroundImageFn() void {
 	if(game.world == null) return;
+	const showItem = itemdrop.ItemDisplayManager.showItem;
+	itemdrop.ItemDisplayManager.showItem = false;
 	renderer.MenuBackGround.takeBackgroundImage();
+	itemdrop.ItemDisplayManager.showItem = showItem;
 }
 fn toggleHideGui() void {
-	gui.hideGui = !gui.hideGui;
+	if(itemdrop.ItemDisplayManager.showItem and !gui.hideGui) {
+		gui.hideGui = true;
+	} else if(itemdrop.ItemDisplayManager.showItem and gui.hideGui) {
+		itemdrop.ItemDisplayManager.showItem = false;
+	} else if(!itemdrop.ItemDisplayManager.showItem and gui.hideGui) {
+		gui.hideGui = false;
+	} else {
+		itemdrop.ItemDisplayManager.showItem = true;
+	}
 }
```

Comment:
I don't think we should have the inventory but no hand item option. Cycling through so many options is kind of annoying.

Reply:
How about shift+F1? F2 is used in screenshots for minecraft, so I would like to preserve it for the same purpose.